docs/get-started/getting-started-react.mdx

```diff
@@ -0,0 +1,381 @@
+
+  APP_CLIENT_ID=
+  APP_CLIENT_SECRET=
+  ```
+
+--- 
+
+### API token variable
+
+In this step, you'll continue to work in the Admin Console to create an API token for your tenant. Then you'll add to the .env.local config file. 
+
+1. In the Admin Console, switch to the **Beyond Identity Admin** realm, select **Apps > Beyond Identity Management API**.
+
+  ![Admin Console Beyond Identity Admin realm](../images/apps-beyond-identity-management-api.png)
+
+2. Select the **API Tokens** tab and click **Create a token**. 
+
+3. Enter a name for the token and click **Create token**.
+
+4. In the confirmation message, copy the API token and click **Close this message**. 
+
+  :::caution IMPORTANT
+  You must copy the API token because after you close the confirmation message, the full token will not be displayed again. You'll only have access to the token suffix.
+  :::
+
+5. In the `.env.local` config file, paste the `API_TOKEN`. Then save your changes but don't close it becuase you'll need it for the next step.
+
+  ```text title=".env.local"
+  # -- Next Auth
+  NEXTAUTH_URL=http://localhost:8083
+  NEXTAUTH_SECRET= # Linux: `openssl rand -hex 32` or go to https://generate-secret.now.sh/32
+
+  # -- Beyond Identity
+  REGION=
+
+  TENANT_ID=
+  REALM_ID=
+  // highlight-next-line
+  API_TOKEN=
+
+  APPLICATION_ID=
+  AUTHENTICATOR_CONFIG_ID=
+
+  APP_CLIENT_ID=
+  APP_CLIENT_SECRET=
+  ```
+
+
+
+---
+
+:::tip **CHECKPOINT**
+
+Guess what? You're halfway there! So far in your getting started journey, you should have completed the following:
+
+- [x] Signed up for a free Beyond Identity developer account, which created a passkey, tenant, and Admin Realm for you automatically.
+
+- [x] Cloned the example application.
+
+- [x] Updated the .env.local config file with your **Region**, **Tenant ID**, and **API token** variables. 
+
+    In the next step, you'll set the following variables:
+    
+    - Realm ID
+    - Application ID
+    - Client ID
+    - Client Secret
+    - Authenticator Config ID<br /><br />
+
+Don't forget, if you get stuck, let us know in our **[Slack community](https://join.slack.com/t/byndid/shared_invite/zt-1anns8n83-NQX4JvW7coi9dksADxgeBQ)**.
+
+:::
+
+---
+
+## 4. Configure Beyond Identity
+
+In this step, you'll create a set of Beyond Identity resources containing your end users' identities and the configuration. First, you'll create a new realm, which holds your users' identities. Next, you'll create a new application that contains the configuration for your users.
+
+### Create a new realm
+
+So far, you have the Beyond Identity Admin Realm for your Beyond Identity tenant only. In this step, you'll create a new Realm to hold your users' identities.  
+
+import CreateRealmConsole from '../includes/_create-realm-console.mdx';
+
+<CreateRealmConsole />
+
+4. Copy the **Realm Id** value and paste it into the `.env.local` file in `REALM_ID`.  Then your changes but don't close it yet because you're not done.
```

Comment:
```suggestion
4. Copy the **Realm Id** value and paste it into the `.env.local` file in `REALM_ID`. Then save your changes but don't close it yet because you're not done.
```